seismic-app/README.md

```diff
@@ -37,6 +37,12 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+### `docker-compose up -d --build`
```

Comment:
@redecs could you give me a tip here?
I'm not sure if this is the command people should use each time they need to start the container

Reply:
The command it's correct and it's the safe version.
Now for the nuances:
1. `-d` sends everything to background - I prefer it that way, but some people prefer to run it in foreground so they can see the logs. I prefer to use `docker-compose logs -f <service_name>`.
2. `--build` it's implied if you never started the container(s) before, but it's a good idea to include it every time, since this will rebuild the images (parts of it more specifically), if there are changes in the `Dockerfile`. If you leave this out you can start your project faster (when you already have the images built), but you might have an outdated setup, since you might be running a image based on an older `Dockerfile`.

Reply:
OK, I'll leave it this way, then. Thanks for the explanation!